app/main.py

```diff
@@ -1,7 +1,20 @@
+import json
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+BASE_URL = "https://api.weatherapi.com/v1/current.json?"
+API_KEY = os.getenv("API_KEY")
+CITY = "PARIS"
+print(API_KEY)
 def get_weather() -> None:
-    # write your code here
-    pass
 
+    url = BASE_URL + f"q={CITY}&key={API_KEY}"
+    response = requests.get(url).json()
```

Comment:
It's a good practice to check the response status code before attempting to parse the JSON. This will help handle cases where the API request fails or returns an error. Consider adding a check like `if response.status_code == 200:` before parsing the JSON.